superset/daos/dataset.py

```diff
@@ -99,11 +99,15 @@ def validate_uniqueness(
 
     @staticmethod
     def validate_update_uniqueness(
-        database_id: int, dataset_id: int, name: str
+        database_id: int,
+        schema: str | None,
```

Comment:
what happens if schema is NONE

Reply:
nice! can you add a test to verify if schema is missing

Reply:
Done!